common-services.yml

```diff
@@ -1,4 +1,13 @@
 services:
+  core:
+    image: audius/core:${TAG:-current}
```

Comment:
Since it's under development do we want this to be edge or latest?

Reply:
$TAG will handle it (audius-d sets it appropriately)